src/components/ZetkinTag.tsx

```diff
@@ -0,0 +1,19 @@
+import { ZetkinTag as ZetkinTagType } from 'types/zetkin';
+import { Chip, ChipProps, Tooltip } from '@material-ui/core';
+
+const ZetkinTag: React.FunctionComponent<{
```

Comment:
I made a component in `organize/TagManager` called `TagChip` which I think should be used wherever this is used. Mine doesn't use `Chip` but I realise now this is a good idea so will do that in my next PR.